src/noit_metric_director.c

```diff
@@ -640,11 +796,19 @@ void noit_metric_director_init() {
   nthreads = eventer_loop_concurrency();
   mtevAssert(nthreads > 0);
   queues = calloc(sizeof(*queues),nthreads);
+  search_asts = calloc(sizeof(*search_asts),nthreads);
+  for(int i=0; i<nthreads; i++) {
+    ck_spinlock_init(&search_asts[i].lock);
+    ck_hs_init(&search_asts[i].hs, CK_HS_MODE_OBJECT | CK_HS_MODE_SPMC,
+               search2_hash, search2_compare,
+               &mtev_memory_safe_ck_malloc, 128, mtev_rand());
+  }
 
   mtev_memory_begin();
   check_interests = mtev_memory_safe_calloc(sizeof(*check_interests),nthreads);
   mtev_memory_end();
   pthread_mutex_init(&check_interests_lock, NULL);
+  pthread_mutex_init(&id_level_lock, NULL);
```

Comment:
`id_level_lock` is no longer used anywhere except here

Reply:
removed